### ISSUE TITLE
Jinshi-Ben-Meeting 2021-06-29

### ISSUE THREAD

**Issue by jinshijian** [PNNL-TES/TES-mgmt#62]
## Global C
* GPP from those sites have similar GPP coverage, however, lack sites for low GPP areas
* Rroot:RA showed a significant relationship with GPP, but Rrrot:Rs and RA:GPP showed no significant relationship with GPP
![FigureSX](https://user-images.githubusercontent.com/13302161/123741656-03ea7080-d8dd-11eb-8df2-c3e7dae0db3e.png)
* I developed a new Table 2
![image](https://user-images.githubusercontent.com/13302161/123753616-a2320280-d8ec-11eb-811d-cf0ff0ef7624.png)

* Another argument presented in the case of Rs:GPP ratio explicitly given in the SRDB database or estimated from combination of SRDB with FLUXNET is to explain that there is probably no bias related to spatial extend of data since weighting (or not) the measurements by vegetation areas globally gives similar results. But it just proves that values from underrepresented vegetation type is not very different for those that are overestimated. But if sites on vegetation from which there is only few measurements are not well representative of the whole ecosystem we can still have large biases. We can also notice that if GPP is estimated from the in-situ Rs:GPP ratio of 0.54 and the new global Rs estimate of 93 GtC we find a global GPP value of 172 GtC which is even larger that estimation given from GPP_Rslit!
* About site-level data it is mentioned that FLUXNET GPP was linked to an SRBD Rs if both measurements occurred within 5km in the same vegetation type and the same year. However, as I mentioned previously, there is another criteria that should be also taken into account which is the land use history of the site. Indeed if a land conversion occurred in the last decades, Rs will not be in equilibrium with GPP making the Rs:GPP ratio incorrect. This is obviously a difficult question as the site history is not necessarily reported. The fact that the estimated Rs:GPP ratio estimated is similar to value reported in the SRDB data (for which I guess both Rs and GPP are reported from the same site) seems to indicate that this not induce an important bias but this point should be considered carefully and discussed however.

## Land models
* Accepted

**Comment by jinshijian** [comment]
![image](https://user-images.githubusercontent.com/13302161/123964593-25834f00-d9e6-11eb-91a4-7c0730466991.png)
Fig. 5: Interesting that the variance seems higher with GSWP3 than with CRU NCEP any thoughts on this added in the discussion would be welcomed.

**Comment by bpbond** [comment]
>But if sites on vegetation from which there is only few measurements are not well representative of the whole ecosystem we can still have large biases

This is true—the fact that weighting measurements by vegetation areas globally gives similar results isn't dispositive. As the reviewer points out, one can construct scenarios where biases in sampling cause serious biases in global estimates. Nonetheless, we think that our weighting test is useful and strongly argues against one category of possible bias. We have modified the text to more clearly make this point.

**Comment by bpbond** [comment]
>However, as I mentioned previously, there is another criteria that should be also taken into account which is the land use history of the site. Indeed if a land conversion occurred in the last decades, Rs will not be in equilibrium with GPP making the Rs:GPP ratio incorrect. This is obviously a difficult question as the site history is not necessarily reported. The fact that the estimated Rs:GPP ratio estimated is similar to value reported in the SRDB data (for which I guess both Rs and GPP are reported from the same site) seems to indicate that this not induce an important bias but this point should be considered carefully and discussed however.

We agree with the reviewer: site history could have a confounding effect, and it's not consistently reported. However, (1) FLUXNET sites tend to be located in undisturbed ecosystems (there are exceptions of course, but overall we believe this is true); and (2) an site-history effect would create more noise in the relationship, but wouldn't necessarily induce a bias. As in our previous response, we acknowledge that we cannot conclusively prove there isn't bias here, but we believe it's quite unlikely. This issue is now discussed in the manuscript (lines XXX).

**Comment by bpbond** [comment]
* In simulation heterotrophic respiration (RH), the microbially-explicit CORPSE and MIMICS did not measurably improve performance compared with CASA-CNP .
* The models were highly sensitive to the input data used to drive them. The quality of such forcing data is critical to improve in the future.
